src/components/HistoryTable/HistoryTableIcon/index.tsx

```diff
@@ -39,15 +36,16 @@ export const HistoryTableIcon: React.FC<HistoryTableIconProps> = ({
         triggerEvent="hover"
       >
         <TestNameContainer>
-          {failingTests.map(({ testName, testId }) => (
-            <Body key={testId}>{testName}</Body>
+          {failingTests.map((testName) => (
+            <TestName key={testName}>{testName}</TestName>
           ))}
+          {loadingTestResults && <Skeleton active data-cy="skeleton" />}
         </TestNameContainer>
       </Tooltip>
     )}
   >
     <Container onClick={() => onClick()} data-cy="history-table-icon">
```

Comment:
This line is throwing an error for me when run from http://localhost:3000/variant-history/evergreen/ubuntu1604?tasks=docker-cleanup using `yarn prod`. Since `onClick` is an optional prop perhaps we want to provide a default noop?

```
Uncaught TypeError: onClick is not a function
    onClick index.tsx:47
    React 14
    unstable_runWithPriority scheduler.development.js:646
    React 4
index.tsx:47
```

https://user-images.githubusercontent.com/9298431/146421282-fcc19013-a37c-4a79-b85e-03a372c6c8e8.mov

Reply:
Nice catch